Packs/ExportIndicators/Integrations/ExportIndicators/ExportIndicators.py

```diff
@@ -64,14 +73,16 @@ def get_params_port(params: dict = demisto.params()) -> int:
     return port
 
 
-def refresh_outbound_context(indicator_query: str, out_format: str, limit: int = 0) -> str:
+def refresh_outbound_context(indicator_query: str, out_format: str, limit: int = 0, mgw_type="string",
```

Comment:
please fix this mwg typo, and every other part in the code that uses it

Reply:
done